installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java

```diff
@@ -66,13 +73,21 @@ private long getInstallationSize()
          Pack p = (Pack) iter.next();
          totalSize += p.nbytes;
       }
-     
+      
+      // Account for any additional features in the total aggregate size.
+      String addFeatureSizes = idata.getVariable("INSTALL_ADDITIONAL_SIZES");
```

Comment:
This two blocks look alike. Would be good to ether concatenate sizes and then do all in on cycle or create static method for that cycle.